pkgs/top-level/all-packages.nix

```diff
@@ -32581,17 +32555,9 @@ with pkgs;
 
   jwm-settings-manager = callPackage ../applications/window-managers/jwm/jwm-settings-manager.nix { };
 
-  k3s_1_24 = callPackage ../applications/networking/cluster/k3s/1_24 {
-    buildGoModule = buildGo120Module;
-  };
-  k3s_1_25 = callPackage ../applications/networking/cluster/k3s/1_25 {
-    buildGoModule = buildGo120Module;
-  };
-  inherit (callPackage ../applications/networking/cluster/k3s {
-    buildGoModule = buildGo120Module;
-  }) k3s_1_26 k3s_1_27 k3s_1_28;
-  inherit (callPackage ../applications/networking/cluster/k3s { }) k3s_1_29;
-
+  k3s_1_24 = callPackage ../applications/networking/cluster/k3s/1_24 { };
+  k3s_1_25 = callPackage ../applications/networking/cluster/k3s/1_25 { };
+  inherit (callPackage ../applications/networking/cluster/k3s { }) k3s_1_29 k3s_1_26 k3s_1_27 k3s_1_28;
```

Comment:
```suggestion
  inherit (callPackage ../applications/networking/cluster/k3s {
    buildGoModule = buildGo121Module;
  }) k3s_1_26 k3s_1_27 k3s_1_28 k3s_1_29;
```

Reply:
As I had to bump K3s 1.26, 1.27 to use the newer go compiler version.

Opened a PR fixing it here: https://github.com/NixOS/nixpkgs/pull/296155

No need to make changes to K3s just a rebase after PR is merged.